### PR TITLE
[css-ui-3] Cursor test udpate

### PR DESCRIPTION
**Pull request by frivoal**
This adjusts and completes the test suite, reflecting the changes resolved in https://github.com/w3c/csswg-drafts/issues/1598#issuecomment-337653861

(Also fixes a nearby test description while we're at it).

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-10-19 06:51:15
Finished: 2017-10-19 07:00:05


<h2>Failing Jobs</h2>
<ul>

<li>build_css</li>

<li>chrome:unstable</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19671)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/289831140)

File: css/css-ui-3/cursor-auto-007.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<title>CSS Basic User Interface Test: cursor:auto on unselectable text</title>
+<link rel="author" title="Florian Rivoal" href="https://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#cursor">
+<link rel="help" href="https://drafts.csswg.org/css-ui-4/#content-selection">
```
Comment (mrego) [inline css/css-ui-3/cursor-auto-007.html]: Just a question, this is linking to `css-ui-4` but this folder is `css-ui-3`.
I think the current agreement is that we'll end up having only one folder for each spec,
but letting you know just to be sure this is not a mistake.

File: css/css-ui-3/cursor-auto-007.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+}
+:root {
+  cursor: help; /* give the root element a different cursor so that
+  it is easy to tell if something changes when hovering the target.*/
+}
+
+/* Hide the text if we cannot make it unselectable.
+ user-select is not part of css-ui level 3, so we should not depend on it
+ but it is nice to test it if it is supported.
+ Test for level 4 can take the conditional out. */
```
Comment (mrego) [inline css/css-ui-3/cursor-auto-007.html]: Ok, I guess this explains that the link to `css-ui-4` was done in purpose. :wink:

File: css/css-ui-3/cursor-auto-007.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+.unselectable { display: none; }
+@supports (user-select: none) or (-webkit-user-select: none) {
+  .unselectable {
+    display: block;
+    user-select: none;
+    -webkit-user-select: none; /* Yes, vendor prefixes are ugly. But this one was grandfathered in and support is required by spec. */
```
Comment (mrego) [inline css/css-ui-3/cursor-auto-007.html]: Nit: I usually see vendor prefix before the actual property like:
```css
  -webkit-user-select: none;
  user-select: none;
```

File: css/css-ui-3/cursor-auto-007.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  }
+}
+
+</style>
+<body>
+  <p>The test passes if, when moved over every element inside the blue box, the cursor remains the platform-dependent default cursor.</p>
+  <p>Place the cursor into the orange box for a reference of what this should look like.</p>
+  <div id=test>
+    <!-- text in a button is either not considered text because it's in a replaced element,
+      or it is text but it is unselectable, eitherway, we should get the default cursor-->
```
Comment (mrego) [inline css/css-ui-3/cursor-auto-007.html]: Super-nit: Dunno if there are any guidelines about this, but the rest of the comments in the test are full sentences (starting uppercase and finishing with a dot). Maybe you want to do the same here.

**Review by mrego — APPROVED** [review]
LGTM, just some minor comments inline feel free to ignore them.

File: css/css-ui-3/cursor-auto-007.html (hunk 1 above)
Comment (frivoal) [reply css/css-ui-3/cursor-auto-007.html]: This is primarily a test about css-ui-3, so it is in the level 3 directory, and has a link to level 3. But it does (optionally, thanks to `@supports`) depend on a level 4 feature, so I'm linking to that too.

File: css/css-ui-3/cursor-auto-007.html (hunk 2 above)
Comment (frivoal) [reply css/css-ui-3/cursor-auto-007.html]: Right.

File: css/css-ui-3/cursor-auto-007.html (hunk 4 above)
Comment (frivoal) [reply css/css-ui-3/cursor-auto-007.html]: I'll be careful next time. But I don't think that's worth another round trip through the review system, so the world will have to live with my sub-par capitalization.

**Pull request merged**